nf_core/lint.py

```diff
@@ -286,6 +291,43 @@ def check_readme(self):
                 self.failed.append((6, "Found a bioconda environment.yml file but no badge in the README"))
 
 
+    def check_version_consistency(self):
+        """ Check that the version variable value is numeric """
+        logging.debug('Checking if version is numeric')
+        versions = {}
+        # Get the version definitions
+        # First, from the TRAVIS_TAG env var
+        versions['travis_tag'] = os.environ.get('TRAVIS_TAG')
```

Comment:
So I think that this is a nice bonus check if `TRAVIS_TAG` is defined 👍 

But it shouldn't fail if we're not running on a release tag, it should just be skipped. Otherwise we can only run these tests on a tag, which is too late.

Reply:
eg.
```python
if os.environ.get('TRAVIS_TAG'):
    versions['travis_tag'] = os.environ['TRAVIS_TAG']
```

Reply:
:+1: